app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
-end
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
Use [Rails Internationalization (I18n) API](http://guides.rubyonrails.org/i18n.html) for multi-language support